modules/sratools/prefetch/main.nf

```diff
@@ -0,0 +1,38 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process SRATOOLS_PREFETCH {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
```

Comment:
```suggestion
        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:[:], publish_by_meta:[]) }
```

Reply:
Adding this change causes the linting to fail.

```
'meta:meta' unspecified in saveAs function
'publish_by_meta:['id']' unspecified in saveAs function
```

Reply:
Ok. I thought it didn't make sense here to use `meta.id` (sample name) to publish the results because it should really be published under `id` if anything. But I also remember asking for that lint test to be added in :) 

Can you change it back please?